rstcheck.py

```diff
@@ -379,24 +341,9 @@ def unknown_departure(self, node):
 
 def beginning_of_code_block(node, full_contents):
     """Return line number of beginning of code block."""
-    lines = full_contents.splitlines()
     line_number = node.line
-    code_block_length = len(node.rawsource.splitlines())
```

Comment:
I'm not sure I understand these removals. Could this be  the reason for the problems you [mention](https://github.com/myint/rstcheck/pull/6#issuecomment-75257957).